src/components/ApplicationTile/index.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+import Text from '../Text';
+import styles from './styles.css';
+import getFormattedTimeDifference from '~/utils/getFormattedTimeDifference';
+
+interface Props {
+  companyName: string;
+  roleName: string;
+  status: 'sent' | 'inProgress' | 'rejected' | 'accepted' | 'hired' | 'archived';
+  salary: {
+    from: number;
+    to: number;
+  };
+  linkToOffer: string;
+  createdAt: Date;
+  className?: string;
+}
+
+const ApplicationTile = ({
+  companyName,
+  roleName,
+  status,
+  salary,
+  linkToOffer,
+  createdAt,
+  className,
+}: Props) => {
+  const { t } = useTranslation('applications');
```

Comment:
maybe you could add strong typing for 'applications'? Maybe useTranslation should have defined type `type Namespaces = 'applications' | 'common'` and should be used with `useTranslation` as `useTranslation<Namespaces>('application')`? Maybe you should create separated hook useTranslate to be no need to define `<Namespaces>` in every use of useTranslation hook? :)

Reply:
Ok, I created useI18n hook (other name to not import native library by mistake) which wraps useTranslation. I was able to do it in a lazy way (see useI18n.ts file) so all the props should match the original hook.